spec/requests/partner_users_requests_spec.rb

```diff
@@ -15,10 +15,21 @@
 
   describe "GET #index" do
     it "renders the index template and assigns @users" do
+      user.add_role(:org_admin)
+
       get partner_users_path(default_params.merge(partner_id: partner))
       expect(response).to render_template(:index)
       expect(assigns(:users)).to eq(partner.users)
     end
+
+    context "when user is not authorized" do
+      it "does not have access to index template" do
+        user.add_role(:org_user)
```

Comment:
This gets added automatically, you don't need to do it.

Reply:
Fixed it here [129a2b8](https://github.com/rubyforgood/human-essentials/pull/4468/commits/129a2b83d41257311ae379f3b2ebed7a75537659)